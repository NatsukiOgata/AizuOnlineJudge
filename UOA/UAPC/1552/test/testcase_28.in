1000 8 12 18 8 0 17 19 16 3 8 15 17 17 18 12 2 8 19 2 18 7 11 16 1 2 9 18 19 13 3 15 7 5 11 8 10 19 0 17 8 16 17 14 9 10 11 7 14 17 6 19 11 5 3 14 5 16 18 7 7 6 5 19 16 17 0 19 16 16 18 8 1 11 10 4 14 19 14 15 15 16 3 12 12 17 8 5 10 9 0 0 3 9 13 10 11 19 1 7 10 15 17 12 13 17 15 11 0 8 1 19 16 6 2 11 7 14 16 11 16 0 4 8 9 16 3 11 10 12 0 0 1 7 11 14 13 2 17 7 16 3 7 13 12 10 18 16 11 13 8 10 12 16 3 9 3 3 0 2 15 6 6 2 11 8 1 7 16 15 19 17 9 15 15 8 7 11 8 12 16 18 12 10 4 7 4 14 10 3 3 0 2 12 12 10 15 10 5 15 7 0 4 8 12 15 17 15 15 11 3 13 13 5 7 5 19 13 7 3 6 4 16 3 7 10 5 3 19 18 0 10 0 8 2 16 4 6 1 19 10 8 11 3 15 5 16 14 5 9 5 18 16 10 13 1 16 7 5 6 17 2 19 9 12 10 1 15 8 9 16 6 2 0 13 1 11 1 2 7 5 2 18 18 19 19 18 11 8 14 10 4 15 3 13 18 10 2 7 10 8 2 11 17 19 3 13 0 2 10 12 4 14 17 19 2 13 10 13 18 5 17 1 0 18 5 14 4 11 6 14 14 18 16 9 2 13 8 14 19 6 8 6 6 4 7 7 7 18 7 0 19 11 6 15 0 17 2 0 2 16 1 18 5 3 13 4 11 15 13 6 12 12 0 9 12 7 3 18 0 17 3 19 18 1 15 13 17 9 6 15 14 10 19 15 9 18 10 16 0 2 4 4 2 0 11 11 14 12 9 8 0 12 12 12 19 14 4 12 11 13 11 15 9 11 18 5 13 14 16 11 19 19 10 9 2 11 6 13 14 17 14 16 16 19 6 11 11 11 13 18 0 6 9 10 18 9 4 17 10 15 5 18 8 8 9 6 15 11 12 12 5 6 1 7 5 6 12 13 14 4 3 19 9 9 1 13 3 1 5 19 6 1 13 3 5 9 10 6 18 15 12 1 16 16 16 16 3 17 5 5 0 3 6 4 17 0 8 7 17 16 6 13 9 5 13 9 13 14 11 5 2 0 8 13 16 5 1 9 6 6 0 10 8 14 13 7 0 19 4 14 11 0 14 19 2 14 11 10 8 8 3 7 13 16 14 12 4 0 17 15 18 2 19 6 11 0 15 18 18 3 2 10 8 0 5 15 15 4 18 7 15 6 19 13 7 1 3 18 11 15 4 10 12 18 10 18 6 11 12 1 3 5 5 3 8 1 2 17 3 8 1 15 5 13 16 8 4 3 9 11 13 11 4 0 2 8 19 11 19 16 19 5 13 8 17 12 8 18 16 9 8 8 8 6 3 16 2 12 3 17 4 2 0 18 15 6 17 10 4 11 0 13 5 7 17 12 19 4 0 17 12 2 5 18 11 11 15 18 7 16 17 19 9 17 16 9 14 2 14 7 16 6 15 5 9 7 19 4 16 16 18 13 9 18 0 18 14 4 5 17 2 5 3 4 2 6 17 11 3 2 19 5 5 3 5 18 7 8 6 11 10 5 2 11 11 1 6 8 9 6 13 16 2 8 13 11 17 17 10 17 16 3 0 7 7 0 16 10 19 16 16 7 18 14 2 11 4 12 8 0 17 18 12 4 8 13 2 4 1 18 15 2 4 13 19 4 16 8 17 0 10 8 15 19 2 15 7 17 17 19 10 9 4 1 18 2 8 2 0 6 2 5 19 4 15 15 1 7 2 6 2 2 14 1 9 11 8 4 12 1 3 5 8 4 12 5 9 16 13 12 16 8 8 7 2 11 9 15 12 6 17 5 6 17 19 4 4 11 0 16 8 14 0 15 10 0 0 8 12 9 12 18 7 8 15 0 6 1 13 13 13 12 14 0 12 12 16 15 0 11 8 5 2 2 3 4 9 12 5 4 10 9 1 4 7 18 9 4 17 19 3 17 9 5 16 10 11 13 7 14 14 16 0 6 11 13 11 16 6 12 10 16 5 10 4 18 19 6 17 8 5 7 14 19 16 16 6 3 4 18 3 4 15 17 4 10 7 17 11 9 13 19 12 16 16 5 7 3 12 15 8 6 8 14 0 18 13 11 0 1 3 12 11 3 17 6 2 5 15 9
