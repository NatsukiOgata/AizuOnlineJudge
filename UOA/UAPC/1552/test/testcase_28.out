18 9 29 294 302
